frontend/smart-kickers-game/src/components/Game/GameStatistics.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import './GameStatistics.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Button } from '../Button/Button.js';
+
+function GameStatistics({ finalScores, handleEndGame, handleResetGame }) {
+  function handleNewGame() {
+    handleEndGame();
+  }
+  return (
+    <>
+      <h2>
+        <em>Statistics</em>
+      </h2>
+      <table>
```

Comment:
Consider using the grid instead of table, grid layout is more readable and simpler than a table element.

Reply:
Ok, will be changed later

Reply:
Done. No more html table.